src/app/models/users.go

```diff
@@ -38,10 +38,12 @@ func (User) FetchQuery() string {
 }
 
 func (u *User) Create(ctx context.Context) error {
+
+	passwordExpired := u.Password == nil
 	rows, err := database.Query(
 		ctx,
 		"users/register",
-		u.FirstName, u.LastName, u.Username, u.Email, u.Password,
+		u.FirstName, u.LastName, u.Username, u.Email, u.Password, passwordExpired,
```

Comment:
no no you don't need to pass password Expired to create user on create query sql if password is null you may set password_expired true on sql query without any parameter